operations/masterbosh-ntp.yml

```diff
@@ -2,21 +2,23 @@
 - type: replace
   path: /resource_pools/0/env/bosh/ntp?
   value:
-    - server time-a.nist.gov
-    - server time-b.nist.gov
-    - server time-c.nist.gov
-    - server time-d.nist.gov
-    - server time-b-wwv.nist.gov
-    - server time-c-wwv.nist.gov
-    - server time-d-wwv.nist.gov
+    - 169.254.169.123
```

Comment:
This IP is for AWS TimeSync, I assume. Do we need to keep all the references to NIST timeservers? What is the effect of having both?